EpiAware/docs/src/examples/getting_started.jl

```diff
@@ -236,42 +240,61 @@ However, we now treat the generated data as `truth_data` and make inference with
 We do the inference by MCMC/NUTS using the `Turing` NUTS sampler with default warm-up steps.
 "
 
-# ╔═╡ c8ce0d46-a160-4c40-a055-69b3d10d1770
-truth_data = generated_obs
+# ╔═╡ 7e48a4c5-cd30-4377-8a98-e0c23f2dc31e
+truth_data = random_epidemic.y_t
```

Comment:
this all looks like it just needs to be revert to what is on main?

Reply:
Yes.

Reply:
I've done that.